cmd/kubeadm/app/phases/etcd/local.go

```diff
@@ -81,6 +81,36 @@ func CheckLocalEtcdClusterStatus(client clientset.Interface, cfg *kubeadmapi.Clu
 	return nil
 }
 
+// RemoveStackedEtcdMemberFromCluster will remove local etcd member from etcd cluster,
+// when reset the control plane node.
+func RemoveStackedEtcdMemberFromCluster(client clientset.Interface, cfg *kubeadmapi.InitConfiguration) error {
+	// creates an etcd client that connects to all the local/stacked etcd members
+	klog.V(1).Info("[etcd] creating etcd client that connects to etcd pods")
+	etcdClient, err := etcdutil.NewFromCluster(client, cfg.CertificatesDir)
+	if err != nil {
+		return err
+	}
+
+	// notifies the other members of the etcd cluster about the removing member
+	etcdPeerAddress := etcdutil.GetPeerURL(&cfg.LocalAPIEndpoint)
+
+	klog.V(2).Infof("[etcd] get the member id from peer: %s", etcdPeerAddress)
+	id, err := etcdClient.GetMemberID(etcdPeerAddress)
+	if err != nil {
+		return err
+	}
+
+	klog.V(1).Infof("[etcd] Removing etcd member: %s, id: %v", etcdPeerAddress, id)
```

Comment:
let's use `%d` for uint64 instead of `%v`
`Removing` -> `removing` to keep consistent the lower case in this file.

Reply:
Done.